Tasks/JenkinsDownloadArtifactsV1/package.json

```diff
@@ -8,15 +8,16 @@
   "author": "Microsoft Corporation",
   "license": "MIT",
   "dependencies": {
-    "@types/node": "^10.17.0",
     "@types/mocha": "^5.2.7",
+    "@types/node": "^10.17.0",
     "@types/uuid": "^8.3.0",
-    "artifact-engine": "0.1.26",
+    "artifact-engine": "1.1.0",
     "azure-pipelines-task-lib": "^3.1.7",
     "azure-pipelines-tasks-azure-arm-rest": "1.0.0",
     "azure-storage": "2.10.4",
     "decompress-zip": "0.3.3",
     "fs-extra": "5.0.0",
+    "handlebars": "^4.7.7",
```

Comment:
Added handlebars to package.json, because after updating artifact-engine to 1.1.0 version, it was removed from package-lock.json as dependency and put into artifact-engine's dependencies.

But we still have some files, where we need handlebars. For example: JenkinsRestClient.ts